build.gradle

```diff
@@ -73,19 +75,24 @@ static def rustTargetTriple() {
     System.getenv("JNILIB_RUST_TARGET")
 }
 
-processResources {
+sourcesJar {
     dependsOn 'metaProperties', 'copyJniLib'
-    from(new File(project.buildDir, "tmp")) {
-        include 'wasmtime-java-meta.properties'
-    }
-    from(new File(project.buildDir, "jni-libs")) {
-        include "libwasmtime_jni_${project.version}_${jniLibOsClassifier()}.*"
-        include "wasmtime_jni_${project.version}_${jniLibOsClassifier()}.*"
+}
+
+sourceSets {
+    main {
+        resources {
+            srcDirs += [ "build/metaProperties", "build/jni-libs" ]
```

Comment:
I thought these generated resources doesn't match the concept of "sourceSet" so it doesn't make sense to define it in this way?

Reply:
According to my contact at Gradleware this is the accepted tool, and he reviewed my solution. There's an example in the docs that almost matches (https://docs.gradle.org/current/dsl/org.gradle.api.tasks.SourceSetOutput.html second sample down) but isn't quite complete compared to what we have. Further evidence that it is "normal" is that this works in VSCode where the version in master does not.

Reply:
I see, good to know that. Thanks!